src/Uno.UI/UI/Xaml/Media/AcrylicBrush/Android/CustomRealtimeBlurView.cs

```diff
@@ -0,0 +1,431 @@
+			mBlurImpl.Release();
+		}
+
+		protected bool Prepare()
+		{
+			if (mBlurRadius == 0)
+			{
+				Release();
+				return false;
+			}
+
+			float downsampleFactor = mDownsampleFactor;
+			float radius = mBlurRadius / downsampleFactor;
+			if (radius > 25)
+			{
+				downsampleFactor = downsampleFactor * radius / 25;
+				radius = 25;
+			}
+
+			int width = Width;
+			int height = Height;
+
+			int scaledWidth = System.Math.Max(1, (int)(width / downsampleFactor));
+			int scaledHeight = System.Math.Max(1, (int)(height / downsampleFactor));
+
+			bool dirty = mDirty;
+
+			if (mBlurringCanvas == null || mBlurredBitmap == null
+					|| mBlurredBitmap.Width != scaledWidth
+					|| mBlurredBitmap.Height != scaledHeight)
+			{
+				dirty = true;
+				ReleaseBitmap();
+
+				bool r = false;
+				try
+				{
+					mBitmapToBlur = Bitmap.CreateBitmap(scaledWidth, scaledHeight, Bitmap.Config.Argb8888);
+					if (mBitmapToBlur == null)
+					{
+						return false;
+					}
+					mBlurringCanvas = new Canvas(mBitmapToBlur);
+
+					mBlurredBitmap = Bitmap.CreateBitmap(scaledWidth, scaledHeight, Bitmap.Config.Argb8888);
+					if (mBlurredBitmap == null)
+					{
+						return false;
+					}
+
+					r = true;
+				}
+				catch (Java.Lang.OutOfMemoryError)
+				{
+					// Bitmap.createBitmap() may cause OOM error
+					// Simply ignore and fallback
+				}
+				finally
+				{
+					if (!r)
+					{
+						Release();
+					}
+				}
+				if (!r)
+				{
+					return false;
+				}
+			}
+
+			if (dirty)
+			{
+				if (mBlurImpl.Prepare(Context, mBitmapToBlur, radius))
+				{
+					mDirty = false;
+				}
+				else
+				{
+					return false;
+				}
+			}
+
+			return true;
+		}
+
+		protected void Blur(Bitmap bitmapToBlur, Bitmap blurredBitmap)
+		{
+			mBlurImpl.Blur(bitmapToBlur, blurredBitmap);
+		}
+
+		protected View GetActivityDecorView()
+		{
+			Context ctx = Context;
+			for (int i = 0; i < 4 && ctx != null && !(ctx is Activity) && ctx is ContextWrapper; i++)
+			{
+				ctx = ((ContextWrapper)ctx).BaseContext;
+			}
+			if (ctx is Activity)
+			{
+				return ((Activity)ctx).Window.DecorView;
+			}
+			else
+			{
+				return null;
+			}
+		}
+
+		protected override void OnAttachedToWindow()
+		{
+			base.OnAttachedToWindow();
+			mDecorView = GetActivityDecorView();
+			if (mDecorView != null)
+			{
+				mDecorView.ViewTreeObserver.AddOnPreDrawListener(_preDrawListener);
+				mDifferentRoot = mDecorView.RootView != RootView;
+				if (mDifferentRoot)
+				{
+					mDecorView.PostInvalidate();
+				}
+			}
+			else
+			{
+				mDifferentRoot = false;
+			}
+		}
+
+		//@Override
+		protected override void OnDetachedFromWindow()
+		{
+			if (mDecorView != null)
+			{
+				mDecorView.ViewTreeObserver.RemoveOnPreDrawListener(_preDrawListener);
+			}
+			Release();
+			base.OnDetachedFromWindow();
+		}
+
+		//@Override
+		public override void Draw(Canvas canvas)
+		{
+			if (mIsRendering)
+			{
+				// Quit here, don't draw views above me
+				//throw STOP_EXCEPTION;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5932087)